src/main/resources/logback.xml

```diff
@@ -0,0 +1,26 @@
+<!-- https://www.playframework.com/documentation/latest/SettingsLogger -->
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>
+                %date{HH:mm:ss} [%thread] %highlight(%-5level) %cyan(%logger{16}) : %message%n%xThrowable{10}
```

Comment:
Would be good if we log things as JSON so that stack traces are shown as single event for example in logentries. Related: https://github.com/commercetools/commercetools-payment-to-order-processor/issues/4